webrender/src/render_backend.rs

```diff
@@ -1040,6 +1040,10 @@ impl RenderBackend {
 
         debug_assert!(op.render || !op.composite);
 
+        if op.build {
```

Comment:
If I'm understanding the code correctly, this won't actually be seen until the next render anyway.

Perhaps it would be better to store if the scene was built in the render backend, and pass this as a flag during the next render call?